cmd/helm/version_test.go

```diff
@@ -30,8 +30,8 @@ import (
 func TestVersion(t *testing.T) {
 	lver := regexp.QuoteMeta(version.GetVersionProto().SemVer)
 	sver := regexp.QuoteMeta("1.2.3-fakeclient+testonly")
-	clientVersion := fmt.Sprintf("Client: &version\\.Version{SemVer:\"%s\", GitCommit:\"\", GitTreeState:\"\"}\n", lver)
-	serverVersion := fmt.Sprintf("Server: &version\\.Version{SemVer:\"%s\", GitCommit:\"\", GitTreeState:\"\"}\n", sver)
+	clientVersion := fmt.Sprintf("Client: &version\\.Version{SemVer:\"%s\", GitCommit:\"\", GitTreeState:\"\", GoVersion:\"go1.12.5\"}\n", lver)
```

Comment:
embedding runtime/os configuration in testdata is generally considered an anti-pattern. The tests would nee to be updated each time a new version of Go is released. Can you strip this information out? Have a look at https://github.com/helm/helm/pull/5869 for an example.

Reply:
Sure.